java-manta-client-unshaded/src/main/java/com/joyent/manta/config/SettableConfigContext.java

```diff
@@ -180,6 +180,14 @@
      */
     T setSkipDirectoryDepth(Integer depth);
 
+    /**
+     * Sets a limit to the number of parent directories deleted, after a delete.
+     *
+     * @param depth directory depth to assume exists
```

Comment:
Holdover from `setSkipDirectoryDepth`, should say something like `empty parents to prune upon deletion of an object`